code/game/jobs/job_exp.dm

```diff
@@ -0,0 +1,197 @@
+
+// Globals, Defines, Etc
+
+var/global/list/exp_jobsmap = list(
+	"Crew" = list(), // all playtime from all jobs, together
+	"Command" = list(titles = command_positions),
+	"Engineering" = list(titles = engineering_positions),
+	"Security" = list(titles = security_positions),
+	"Silicon" = list(titles = nonhuman_positions),
+	"Service" = list(titles = service_positions),
+	"Medical" = list(titles = medical_positions),
+	"Science" = list(titles = science_positions),
+	"Supply" = list(titles = supply_positions),
+	"Special" = list(), // special_role
+	"Ghost" = list(), // dead/observer
+	"Exempt" = list() // special grandfather setting
+)
+
+
+// Admin Verbs
+
+
+/client/proc/cmd_admin_check_player_exp()	//Allows admins to determine who the newer players are.
+	set category = "Admin"
+	set name = "Check Player Playtime"
+	if(!check_rights(R_ADMIN))
+		return
+	var/msg = "<html><head><title>Playtime Report</title></head><BR>Playtime:<BR>"
+	for(var/client/C in clients)
+		var/list/play_records = params2list(C.prefs.exp)
+		var/gen_exp_time = text2num(play_records["Crew"])
+		gen_exp_time = num2text(round(gen_exp_time / 60)) + "h"
+		msg += "<BR> - [key_name_admin(C.mob)]: <A href='?_src_=holder;getplaytimewindow=\ref[C.mob]'>" + C.get_exp_general() + "</a> "
+	if(msg != "")
+		src << browse(msg, "window=Player_playtime_check")
+	log_admin("[key_name(usr)] checked player playtime")
+	feedback_add_details("admin_verb","MCPP") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+
+
+/client/proc/cmd_show_exp_panel(var/client/C in clients)
+	if(!check_rights(R_ADMIN))
+		return
+	var/body = "<html><head><title>Playtime for [C.key]</title></head><BR>Playtime:"
```

Comment:
Another missing body tag, ironically in a var named `body`. The html tag *does* get closed below, though, so that's nice.